### PR TITLE
Adds a rRaman example, renames and changes data.from_rRaman to data.from_BrunoldrRaman

### LINKED ISSUE #162: Example of Brunold Group rRaman data loading and plotting

### PR DESCRIPTION
**Pull request by neffmallon**
resolves #162, lessens dependence on from_text (see #153)

### CODE REVIEW

File: WrightTools/data.py [hunk 1]
```diff
@@ -1934,6 +1934,62 @@ def from_text(filepath, name=None, verbose=True):
...
+        else:
+            name = filepath.split('//')[-1].split('.')[0]
+            data = Data([axis], [signal], source='txt file', name=name)
+    # finish
+    if verbose:
+        print('{0} data objects successfully created from file:'.format(len(indicies)))
+    return data
+
+
+def from_BrunoldrRaman(filepath, name=None, verbose=True, trash_pixels=56):
```
Comment (ksunden) [inline WrightTools/data.py]: If trash_pixels is a kwarg, it should be listed in the docstring

File: WrightTools/data.py [hunk 2]
```diff
@@ -1934,6 +1934,62 @@ def from_text(filepath, name=None, verbose=True):
...
+    if isinstance(filepath, type([])) or isinstance(filepath, type(np.array([]))):
+        return [from_BrunoldrRaman(f) for f in filepath]
+    if not os.path.isfile(filepath):
+        raise wt_exceptions.FileNotFound(path=filepath)
+    if not filepath.endswith('txt') and not filepath.endswith('dpt'):
+        wt_exceptions.WrongFileTypeWarning.warn(filepath, 'txt or dpt')
+    # import array
+    lines = []
+    with open(filepath, 'r') as f:
+        for i in range(trash_pixels):
```
Comment (ksunden) [inline WrightTools/data.py]: @untzag What is your opinion on doing this kind of data removal in a `from` method?

The particular instrument that the Brunold group has apparently produces junk data at the beginning of the file.  It is not clear what the cause of these junk values is, but they are not valuable to the data object itself

File: WrightTools/data.py [hunk 3]
```diff
@@ -1934,6 +1934,62 @@ def from_text(filepath, name=None, verbose=True):
...
+            line = f.readline()
+        while True:
+            line = f.readline()
+            if line == '\n' or line == '':
+                break
+            else:
+                clean = line[:-2]  # lines end with ',/n'
```
Comment (ksunden) [inline WrightTools/data.py]: The datafile provided does not have commas, are we losing the precision here 

(Admittedly in the example file, the truncated digit would always be zero, but I do not know if the precision shown is always the case)

File: WrightTools/data.py (hunk 2 above)
Comment (neffmallon) [inline WrightTools/data.py]: I went upstairs and talked to the Brunold group and it looks like the negative pixel problem has been either fixed or never existed. I've taken the trash_pixels out of this code and changed the example code too.

**Comment by untzag** [comment]
thanks so much @neffmallon

**Comment by neffmallon** [comment]
old branch is still around, should it be deleted meow?

**Comment by untzag** [comment]
I don't see it @neffmallon

**Pull request merged**